src/app/core/common-components/entity-form/entity-form/entity-form.component.ts

```diff
@@ -88,48 +88,42 @@ export class EntityFormComponent<T extends Entity = Entity>
     }
   }
 
-  private async applyChanges(entity: T) {
-    if (this.formIsUpToDate(entity)) {
-      Object.assign(this.entity, entity as any);
+  private async applyChanges(externallyUpdatedEntity: T) {
+    if (this.formIsUpToDate(externallyUpdatedEntity)) {
+      Object.assign(this.entity, externallyUpdatedEntity as any);
       return;
     }
+
+    const userEditedFields = Object.entries(this.form.getRawValue()).filter(
+      ([key, value]) => this.form.controls[key].dirty,
+    );
+    let userEditsToReapply = userEditedFields.filter(([key, value]) =>
```

Comment:
```suggestion
    let editsWithoutConflict = userEditedFields.filter(([key, value]) =>
```
I had quite a difficult time to understand what this property actually holds.